stable/airflow/values.yaml

```diff
@@ -765,6 +765,20 @@ flower:
   ##
   basicAuthSecret: ""
 
+  oauth:
```

Comment:
lets replace this `oauth` value with a top-level value called `oauthDomains`, as there is no real point in creating `oauth.enabled` when we can just check if the `oauthDomains` value is empty

I imagine the docstring would look like this:
```yaml
## the value of the flower `--auth` argument
##
## NOTE:
##  - see flower docs: https://flower.readthedocs.io/en/latest/auth.html#google-oauth-2-0
##
oauthDomains: ""
```

Reply:
This also means you have to add `flower.oauthDomains` to README.md